projects/kit/components/input-phone-international/test/international-input-phone.component.spec.ts

```diff
@@ -143,6 +143,30 @@ describe('InputPhoneInternational', () => {
             expect(component.countryIsoCode).toBe(TuiCountryIsoCode.UA);
         });
 
+        it('should set country code on paste event ', () => {
+            const data = new DataTransfer();
+
+            data.setData('text/plain', '88005553535');
+
+            const pasteEvent = new ClipboardEvent('paste', {clipboardData: data as any});
+
+            component.onPaste(pasteEvent);
+
+            expect(component.countryIsoCode).toBe(TuiCountryIsoCode.RU);
+        });
+
+        it('should replace code on paste event ', () => {
+            const data = new DataTransfer();
+
+            data.setData('text/plain', '88005553535');
```

Comment:
can we add a test where the situation might be: '+89112223344'

Reply:
done 👍